### PR TITLE
verify static code addition, eslint added

### PR DESCRIPTION
**Pull request by edopore**
Good Afternoon this PR contains code about static code and eslint

### CODE REVIEW

**Comment by edopore** [comment]
Hi, I'm at github auth part, I established my environment variable with my Personal Access Token, unfortunately I found errors about eslint that are related with ident, spaces and endlines before arrow functions, I tried to resolve adding rules to .eslint.yml file.
When I make test to github auth I see that first part pass the test even if I don't change my username and repository data.
But at second test with parameters this test not pass.
I saw at bootcamp session last week that a possible solution is set a varible inside the test code to pass, but I don't remember what it has to do.
ErrorGitHubAuth
.

File: .eslintrc.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  es6: true
+  node: true
+  mocha: true
+extends:
+  - eslint:recommended
+  - airbnb-base
+  - plugin:import/errors
+  - plugin:import/warnings
+rules:
+  "comma-dangle": ["error", "never"]
```
Comment (holgiosalos) [inline .eslintrc.yml]: EOF

**Comment by holgiosalos** [comment]
> Hi, I'm at github auth part, I established my environment variable with my Personal Access Token, unfortunately I found errors about eslint that are related with ident, spaces and endlines before arrow functions, I tried to resolve adding rules to .eslint.yml file. When I make test to github auth I see that first part pass the test even if I don't change my username and repository data. But at second test with parameters this test not pass. I saw at bootcamp session last week that a possible solution is set a varible inside the test code to pass, but I don't remember what it has to do. ErrorGitHubAuth .

Please share the code that is presenting the problem

**Comment by edopore** [comment]
Good afternoon.
The code with error is that part marked
problemtest2

File: .eslintrc.yml (hunk 1 above)
Comment (edopore) [reply .eslintrc.yml]: I fixed EOF witn a new commit

File: .eslintrc.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
```
Comment (holgiosalos) [inline .eslintrc.yml]: To fix the EOF, you should make sure the line 12 is totally empty, without tab or whitespace, the warning is still there:
image

Remember that you can configure your visual studio code to add automatically the EOF: https://stackoverflow.com/a/44704969

**Review by holgiosalos — COMMENTED** [review]
About the issue you are having, why you are using superagent? the guide is configured to use axios:
https://github.com/holgiosalos/workshop-api-testing-js-doc#autenticaci%C3%B3n-en-github

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**